bin/propolis-server/src/lib/migrate/source.rs

```diff
@@ -129,7 +148,7 @@ impl<T: AsyncRead + AsyncWrite + Unpin + Send> SourceProtocol<T> {
 
         loop {
             let m = self.read_msg().await?;
-            info!(self.log(), "ram_push: source xfer phase recvd {:?}", m);
+            trace!(self.log(), "ram_push: source xfer phase recvd {:?}", m);
             match m {
                 codec::Message::MemDone => break,
                 codec::Message::MemFetch(start, end, bits) => {
```

Comment:
(it has now been 0 reviews since I got mad about Github not letting me comment on arbitrary lines of a file in a change)

I think it'd also be good to have a probe somewhere around here (or in `xfer_ram`) that indicates the number of pages we transferred. This should let us compute data transfer rates in the script, which are important context when evaluating whether the `RamPush` phase is taking a long time.

Reply:
Sure, that's something I've wanted to. I'll take a stab at doing so in this PR.